java/ql/test/ext/TopJdkApis/TopJdkApisTest.expected

```diff
@@ -0,0 +1,97 @@
+| java.lang.String#charAt(int) |
+| java.lang.String#contains(CharSequence) |
+| java.lang.String#equals(Object) |
+| java.lang.String#equalsIgnoreCase(String) |
+| java.lang.String#format(String,Object[]) |
+| java.lang.String#hashCode() |
+| java.lang.String#isEmpty() |
+| java.lang.String#length() |
+| java.lang.String#replace(CharSequence,CharSequence) |
+| java.lang.String#split(String) |
+| java.lang.String#startsWith(String) |
+| java.lang.String#substring(int) |
+| java.lang.String#substring(int,int) |
+| java.lang.String#toLowerCase() |
+| java.lang.String#trim() |
+| java.lang.String#valueOf(int) |
+| java.lang.StringBuffer#append(String) |
+| java.lang.StringBuilder#append(Object) |
+| java.lang.StringBuilder#append(String) |
+| java.lang.StringBuilder#append(char) |
+| java.lang.StringBuilder#append(int) |
+| java.lang.StringBuilder#toString() |
+| java.lang.System#currentTimeMillis() |
+| java.lang.Throwable#Throwable(Throwable) |
+| java.lang.Throwable#getMessage() |
+| java.math.BigDecimal#BigDecimal(String) |
+| java.nio.file.Path#resolve(String) |
+| java.sql.ResultSet#getString(String) |
+| java.util.ArrayList#ArrayList(Collection) |
+| java.util.ArrayList#add(Object) |
+| java.util.Arrays#asList(Object[]) |
+| java.util.Collection#size() |
+| java.util.Collection#stream() |
+| java.util.Collections#emptyList() |
+| java.util.Collections#singletonList(Object) |
+| java.util.Entry#getKey() |
+| java.util.Entry#getValue() |
+| java.util.HashMap#get(Object) |
+| java.util.HashMap#put(Object,Object) |
+| java.util.Iterator#hasNext() |
+| java.util.Iterator#next() |
+| java.util.List#add(Object) |
+| java.util.List#addAll(Collection) |
+| java.util.List#contains(Object) |
+| java.util.List#get(int) |
+| java.util.List#isEmpty() |
+| java.util.List#of(Object) |
+| java.util.List#size() |
+| java.util.Map#containsKey(Object) |
+| java.util.Map#entrySet() |
+| java.util.Map#get(Object) |
+| java.util.Map#isEmpty() |
+| java.util.Map#keySet() |
+| java.util.Map#put(Object,Object) |
+| java.util.Map#remove(Object) |
+| java.util.Map#size() |
+| java.util.Map#values() |
+| java.util.Objects#equals(Object,Object) |
+| java.util.Objects#hash(Object[]) |
+| java.util.Objects#requireNonNull(Object,String) |
+| java.util.Optional#empty() |
+| java.util.Optional#get() |
+| java.util.Optional#ifPresent(Consumer) |
+| java.util.Optional#isPresent() |
+| java.util.Optional#map(Function) |
+| java.util.Optional#of(Object) |
+| java.util.Optional#ofNullable(Object) |
+| java.util.Optional#orElse(Object) |
+| java.util.Set#add(Object) |
+| java.util.Set#contains(Object) |
+| java.util.Set#isEmpty() |
+| java.util.Set#size() |
+| java.util.UUID#randomUUID() |
+| java.util.UUID#toString() |
+| java.util.stream.Collectors#toList() |
+| java.util.stream.Stream#filter(Predicate) |
+| java.util.stream.Stream#findFirst() |
+| java.util.stream.Stream#forEach(Consumer) |
+| java.util.stream.Stream#map(Function) |
```

Comment:
There are 97 lines here. Was this meant to include all 100?

Reply:
It should be 97 instead of 100 due to the fact that [`java.util.stream.Stream#collect(Collector)`](https://github.com/github/codeql/pull/11572#discussion_r1040967124), [`java.lang.String#valueOf(Object)`](https://github.com/github/codeql/pull/11572#discussion_r1040978328), and [`java.lang.Throwable#printStackTrace()`](https://github.com/github/codeql/pull/11572#discussion_r1040991878) are not modeled. 
Let me know if I should add a neutral model for any of these instead of leaving them unmodeled.

Reply:
No, I think that's fine as-is. Once the test is restructured as I've suggested, then it should be more clear anyway, as these will be the only results.